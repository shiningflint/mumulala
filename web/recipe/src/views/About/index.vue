<template>
  <div class="about">
    This is the about page
  </div>
</template>

<style lang="scss" scoped>
.about {
  color: red;
}
</style>
