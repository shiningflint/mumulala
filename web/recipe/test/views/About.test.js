import { shallowMount, createLocalVue } from '@vue/test-utils'
import About from 'views/About'
import VueRouter from 'vue-router'

const localVue = createLocalVue()
localVue.use(VueRouter)
const router = new VueRouter()

describe('About', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(About, {
      localVue,
      router
    })
  })

  it('Mounts without error', () => {
    expect(wrapper.vm.$el).toMatchSnapshot()
  })
})
