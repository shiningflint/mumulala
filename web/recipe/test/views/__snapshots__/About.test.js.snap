// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Mounts without error 1`] = `
<div
  class="about"
>
  
  This is the about page

</div>
`;
